roads.mss

```diff
@@ -38,35 +38,41 @@ as well. */
 
 #roads_med[zoom >= 9] {
   [type='motorway'],
-  [type='motorway_link'] {
-    line-color: @motorway-trunk-line;
-    [bicycle='yes'] {
-      line-color: @motorway-trunk-cycle-fill;
-    }
-  }
+  [type='motorway_link'],
   [type='trunk'],
   [type='trunk_link'] {
-      line-color: @motorway-trunk-cycle-fill;
-      [bicycle='no'] {
-        line-color: @motorway-trunk-fill;
-      }
+      line-color: @motorway-trunk-fill;
```

Comment:
Why are we discarding the `bicycle` switch? motorway/motorway_link should not be accessible by bikes by default, but trunk/trunk_link should be: https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access_restrictions#Default.

Reply:
because the test is not correct and different than higher zoom, we just test bicycle, where we should test more, a can_bicycle should be added otherwize